src/.nuspec

```
@@ -0,0 +1,21 @@
+<?xml version="1.0" encoding="utf-8"?>
+<package xmlns="http://schemas.microsoft.com/packaging/2010/07/nuspec.xsd">
+    <metadata>
+        <!-- Required elements-->
+        <id>SwedbankPay.Sdk</id>
+        <description>A client library and SDK for Swedbank Pay's RESTful API Platform</description>
+        <authors>Swedbank Pay</authors>
+        <version>1.1.1</version>
```

Is it possible to not hard-code `version` in the `.nuspec` and instead just provide it on the command line as minted by GitVersion?

---

Yes, already done

---

But the `version` element still needs to be in the `.nuspec` file?

---

Yes, it needs to be there, but it gets overwritten from the CLI. I've tested it locally to ensure it gets overwritten.

---

Is it possible to set the version number to something invalid so version `1.1.1` is not accidentally published anywhere?

```suggestion
        <version>0.0.1-INVALID</version>
```